PokemonGo.RocketAPI.Logic/Logic.cs

```diff
@@ -79,7 +79,7 @@ public static float CalculatePokemonPerfection(PokemonData poke)
             {
                 var xpDifference = GetXPDiff(playerStat.Level);
                 var message =
-                    $"{playerName} | Level {playerStat.Level}: {playerStat.Experience - playerStat.PrevLevelXp - xpDifference}/{playerStat.NextLevelXp - playerStat.PrevLevelXp - xpDifference}XP";
+                    $"{playerName} | Level {playerStat.Level}: {playerStat.Experience - playerStat.PrevLevelXp - xpDifference}/{playerStat.NextLevelXp - playerStat.PrevLevelXp - xpDifference}XP Stardust: {_playerProfile.Profile.Currency.ToArray()[1].Amount}";
```

Comment:
I see no reason why it shouldn't work. Since the profile is updated whenever the task is running. Otherwise the xp shouldn't work either.